src/test/java/com/github/wovnio/wovnjava/Utf8Test.java

```diff
@@ -0,0 +1,72 @@
+package com.github.wovnio.wovnjava;
+
+import static org.junit.Assert.assertNotEquals;
+
+import java.io.UnsupportedEncodingException;
+
+import org.junit.Ignore;
+
+import junit.framework.TestCase;
+
+public class Utf8Test extends TestCase {
+    public void testDefaultEncodingUTF8() {
+        Utf8 unicodeConverter = new Utf8("");
+        String data = new String("Unicøde Sentence　東京 الخط العربي في يونيكود");
+        byte[] rawData = null;
+        try {
+            rawData = data.getBytes("utf-8");
+        } catch (UnsupportedEncodingException e) {
+            System.exit(1);
```

Comment:
do you really need to exit inside of a unit test?

Reply:
I suppose I can rethrow the exception... just thought it might not be that clean...